perllib/FixMyStreet/Cobrand/Gloucester.pm

```diff
@@ -0,0 +1,103 @@
+=head1 NAME
+
+FixMyStreet::Cobrand::Gloucester - code specific to the Gloucester cobrand
+
+=head1 SYNOPSIS
+
+We integrate with Gloucester's Alloy back end.
+
+=head1 DESCRIPTION
+
+=cut
+
+package FixMyStreet::Cobrand::Gloucester;
+use parent 'FixMyStreet::Cobrand::Whitelabel';
+
+use strict;
+use warnings;
+
+use Moo;
+with 'FixMyStreet::Roles::Open311Alloy';
+
+=head2 Defaults
+
+=over 4
+
+=cut
+
+sub council_area_id { '2325' }
+sub council_area { 'Gloucester' }
+sub council_name { 'Gloucester City Council' }
+sub council_url { 'gloucester' }
+
+=item * Users with a gloucester.gov.uk email can always be found in the admin.
+
+=cut
+
+sub admin_user_domain { 'gloucester.gov.uk' }
+
+=item * Allows anonymous reporting
+
+=cut
+
+sub allow_anonymous_reports { 'button' }
```

Comment:
We shouldn't have this function, but have all categories ticked in the admin - then the staff can decide which categories are anonymous rather than it being forced on by this function.